core/src/main/java/org/apache/iceberg/deletes/Deletes.java

```diff
@@ -135,6 +142,17 @@ public static StructLikeSet toEqualitySet(
     }
   }
 
+  static ExecutorService getDeletePosThreadPool() {
```

Comment:
Iceberg doesn't use `get` in method names. You can either omit it or use a more specific verb.

Reply:
Addressed this in recent PR. Moved it to ThreadPools.newDeleteWorkerPool().